index.html

```diff
@@ -4,484 +4,528 @@
               </div>
+            </div>
+            <p class="flying-card__title">Maliha</p>
+            <p class="flying-card__subtitle">UX Designer</p>
+          </div>
 
-              <!-- Second flying card -->
-              <div class="right-wrapper__flying-card second-flying-card">
-                <div class="flying-card__img-block second-img">
-                  <div class="flying-card__circle">
-                    <img src="./src/icons/star.svg" alt="star" />
-                    <p class="flying-card__rating">4.5</p>
-                  </div>
-                </div>
-                <h2 class="flying-card__title">Adam</h2>
-                <p class="flying-card__subtitle">Project Manager</p>
+          <!-- Second flying card -->
+          <div class="graphics__flying-card second-flying-card">
+            <div class="flying-card__img-block second-img">
+              <div class="flying-card__circle">
+                <img src="./src/icons/star.svg" alt="star" />
+                <p class="flying-card__rating">4.5</p>
               </div>
             </div>
+            <p class="flying-card__title">Adam</p>
+            <p class="flying-card__subtitle">Project Manager</p>
           </div>
         </div>
       </section>
-    </header>
 
-    <main>
-      <!-- First section -->
-      <section class="first-section">
-        <div class="container first-section__container">
-          <h2 class="first-section__title">Popular Job Posts</h2>
-          <!-- Cards -->
-          <div class="first-section__cards-wrapper">
+      <!-- Job posts -->
+      <section class="job-posts">
+        <div class="container job-posts__container">
+          <h2 class="job-posts__title title">Popular Job Posts</h2>
+          <!-- card -->
+          <div class="job-posts__card-wrapper">
             <!-- First card -->
-            <div class="first-section__first-card cards">
-              <div class="cards__content-container">
-                <!-- Cards content top -->
-
-                <div class="cards-content__top">
-                  <div class="cards-content__left-side">
-                    <div class="cards-content__cards-circle">
-                      <img src="./src/icons/slack-2.svg" alt="slack-icon" />
-                    </div>
-                    <p class="cards-content__title white-txt">Slack</p>
+            <a href="#" class="card">
+              <!-- card content top -->
+              <div class="card-content__top">
+                <div class="card-content__company-info">
+                  <div class="card-content__card-circle">
+                    <img src="./src/icons/slack-2.svg" alt="slack-icon" />
+                  </div>
+                  <span class="card-content__company-name">Slack</span>
+                </div>
+                <div class="card-content__avatars">
+                  <div class="right-side__first-circle">
+                    <img src="./src/icons/avatar-1.svg" alt="first avatar" />
```

Comment:
```suggestion
                    <img src="./src/icons/avatar-1.svg" alt="avatar" />
```
This image doesn't have an informational role, so you may leave the `alt` field empty or write just "avatar"

Apply this comment to the same places below.